fabfile.py

```diff
@@ -315,6 +331,15 @@ def pull_database_from_platform(c, environment_name):
     cli_exec(
         f"platform db:dump -e {environment_name} -p {PLATFORM_PROJECT_ID} -f {timestamp}.psql -d {LOCAL_DB_DUMP_DIR}"
     )
+    cli_exec(
+        f"sed -i -e 's/beta.nationalarchives.gov.uk\\([[:space:]]\\)443/localhost\\165535/g' {LOCAL_DB_DUMP_DIR}/{timestamp}.psql"
+    )
+    cli_exec(
+        f"sed -i -e 's/develop.tna.dblclk.dev\\([[:space:]]\\)443/localhost\\165535/g' {LOCAL_DB_DUMP_DIR}/{timestamp}.psql"
+    )
+    cli_exec(
+        f"sed -i -e 's/tna.dblclk.dev\\([[:space:]]\\)443/localhost\\165535/g' {LOCAL_DB_DUMP_DIR}/{timestamp}.psql"
+    )
```

Comment:
What's this last bit here doing?

Reply:
This replaces the instance of either `beta.nationalarchives.gov.uk`, `develop.tna.dblclk.dev` and `tna.dblclk.dev` and the port `443` for the site definition (http://localhost:8000/admin/sites/edit/2/) with `localhost` under port `65535` so the API outputs the correct `full_url`s for the new frontend.